pyeudiw/openid4vp/utils.py

```diff
@@ -0,0 +1,25 @@
+from pyeudiw.openid4vp.vp import Vp
+from pyeudiw.openid4vp.vp_sd_jwt import VpSdJwt
+from pyeudiw.jwt.utils import decode_jwt_header
+from pyeudiw.openid4vp.exceptions import VPFormatNotSupported
+
+
+def handle_VP(jwt: str) -> Vp:
```

Comment:
non è un nome di compliant alla PEP, rinominalo in vp_verifier, vp_parser o simile